.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: build
+
+on:
+    push:
+    pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Install Deps.
+        uses: cytopia/shell-command-retry-action@v0.1.6
```

Comment:
I'd rather not do retry, especially for build, it'll just slow down when there's an actual problem. People can always re-kick if they believe there was a transient error.